convex/http.ts

```
@@ -1,9 +1,160 @@
+			const clientSecret = process.env.SLACK_CLIENT_SECRET;
+			const redirectUri = `${siteUrl}/api/import/slack/callback`;
+
+			if (!clientId || !clientSecret) {
+				throw new Error("Slack OAuth not configured");
+			}
+
+			// Create AbortController for timeout handling
+			const controller = new AbortController();
+			const timeoutId = setTimeout(() => controller.abort(), 15000); // 15 second timeout
+
+			let tokenResponse;
```

Biome (2.3.13)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)